webpack/sass/main.scss

```diff
@@ -0,0 +1,170 @@
+//Color Variables
+$pumpkin: #e15803;
+$orange: #FB7928;
+$candy: #EE5B32;
+$jet-black: #333333;
+$grey: rgba(255, 255, 255, 0.6);
+$blue: #00b5da;
+$white: #fff;
+
+//Color Configs
+$brand-theme: $candy;
+$brand-theme-dark: $pumpkin;
+$brand-theme-light: $orange;
+
+$text-color: $jet-black;
+$project-card-theme: $blue;
+
+body {
+  font-family: 'Source Sans Pro', sans-serif;
+  color: $text-color;
+}
+
+h2 {
+  font-weight: 600;
+  padding-bottom: 0.5rem;
+}
+
+.page-content {
+  position: relative;
+  min-height: calc(100vh - 21vh);
+  padding-bottom: 3rem;
+}
+
+.main-header .navbar-brand {
+  margin-right: 2rem;
+  img {
+    width: 250px;
+  }
+  .legend {
+    display: block;
+    font-size: .9rem;
+    letter-spacing: .15rem;
+  }
+}
+
+.footer {
+  bottom: 0;
+  height: 7rem;  /* Footer height */
+}
+
+.card-columns {
+  column-count: 2;
+}
+
+.main-header {
+  background-color: $brand-theme;
+}
+
+.navbar {
+  background: {
+    image: linear-gradient(90deg, $brand-theme, $brand-theme-light , $brand-theme);
+    size: 100% 130px;
+    position-y: bottom;
+  }
+}
+
+.page-title {
+  background-color: $white;
+  border: {
+    bottom-style: solid;
+    color: $brand-theme;
+    width: 10px;
+  }
+  padding-left: 0px;
+  font-weight: 600;
+  color: $text-color;
+}
+
+@mixin style {
+  background-color: $white;
+  border: {
+    left-color: $project-card-theme !important;
+    left-width: 10px;
+  }
+  padding-left: 0px;
+}
+
+.featured-project-card {
+  @include style();
+  min-width: 20rem;
+  max-width: 20rem;
+}
+
+.project-idea-header {
+  @include style();
+  font-size: 1.5em;
+}
+
+.badge-featured-project {
+  background-color: $project-card-theme;
+  color: $white;
+  font-size: 0.8rem;
+}
+
+#back-to-top {
+  cursor: pointer;
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  display: none;
+  background-color: $blue;
+  border: none;
+}
+
+.nav-item, .dropdown-item {
+  font-size: 1.2em;
+  text-align: center;
+}
+
+.dropdown:hover .dropdown-menu {
+  display: block;
+}
+
+.dropdown-menu {
+  margin-top: 0px;
+}
+
+.nav-item:hover {
+  background: $brand-theme-dark !important;
+}
+
+.navbar-dark {
+  .navbar-nav .nav-item .nav-link {
+    color: $white;
+    line-height: 1rem;
+    border-left: 1px solid $grey;
+  }
+  .navbar-toggler {
+    border-color: $white;
+  }
+  .navbar-nav .nav-item.active .nav-link {
```

Comment:
This line should be nested, because up in the code you still have .nav-item elements

Reply:
[https://github.com/creativecommons/creativecommons.github.io-source/pull/31/commits/a48e53bab9ce7f2531c6f3dce5d9ce5c61706f8b](url)

Reply:
https://github.com/creativecommons/creativecommons.github.io-source/pull/31/commits/a48e53bab9ce7f2531c6f3dce5d9ce5c61706f8b